summary_workflows/identification/specification/I_benchmark_specification.md

```diff
@@ -16,14 +16,22 @@ Comparison of PAS predicted from simulated RNA-Seq data with dataset used for si
 1. Poly(A) sites identified based on simulated RNAseq data using the benchmarked tool
 2. Ground truth: Poly(A) sites of transcripts used to simulate RNAseq reads
 
+Calculating proportion of identified PAS assigned to different genomic features:
+
+1. Poly(A) sites identified based on RNAseq data using the benchmarked tool
+2. GTF file with genome annotation
+
```

Comment:
Thanks @SamBryce-Smith, that's really useful. I agree it's difficult to use this metric for benchmarking, especially if, as you said, the tools focus on 3'UTRs for identification. 

Your interpretation was correct, the metric wasn't supposed to be calculated for ground truth. I can see how that would be useful just for visualisation, but I think it wouldn't make much sense for OEB benchmarking.

I can entirely remove this metric from OEB plots and move it to optional plots/visualisations for the paper.

Reply:
Sorry @SamBryce-Smith, I missed your question about window size/distance threshold. So ideally I would prefer to have a wider range (eg from 0 to 200nt), but given limitations of how many separate 2D scatter plots we can reasonably show, I thought that a few values closer to the ground truth might be more informative. I still think it might be useful to plot up to 200nt in a single 2D line plot, so I left it like that in optional plots.

Reply:
Agreed, think it's a good idea to keep every possible step to the optional plots, especially if we can't visualise all the lines (for every window size) on a single plot in OpenEBench!